packages/shared/tailwind.config.js

```diff
@@ -222,6 +222,9 @@ module.exports = {
       minHeight: {
         card: '23.75rem',
       },
+      maxHeight: {
+        card: '23.75rem',
+      },
```

Comment:
I'm not sure cards should ever have a max-h? 
As they can grow based on content

Reply:
Do they?

Reply:
NVM :)